.travis.yml

```diff
@@ -1,54 +1,51 @@
 notifications:
   email: false
-
 sudo: required
-
 services:
-  - docker
-
+- docker
 language: python
-
 python:
-  - 3.7-dev
-
+- 3.7-dev
```

Comment:
I know you didn't just change this but why are we using a dev version?

Reply:
I don't remember, likely that's not necessary anymore.